.development.env

```diff
@@ -1,5 +1,4 @@
 FLASK_APP=application.py
-FLASK_ENV=development
```

Comment:
How come this has been removed?

Reply:
My mistake - took it out for local dev to check logging under different scenarios